test/cli.js

```diff
@@ -290,3 +290,33 @@ test('handles NODE_PATH', function (t) {
 		t.end();
 	});
 });
+
+test('Show `expected` and `actual` with a message', function (t) {
+	execCli('fixture/assert-equal-with-message.js', function (err, stdout, stderr) {
+		t.ok(err);
+
+		var expectedOutput =
+			'\n    \u001b[31mAssertionError: foo\u001b[31m\n' +
+			'\u001b[32m    + expected\u001b[31m\u001b[31m - actual\u001b[31m\n\n' +
+			'\u001b[31m    -3\u001b[31m\n' +
+			'\u001b[32m    +4\u001b[31m';
```

Comment:
We tend to repeat the `chalk.red()` etc calls in the test, it makes it easier to change things later. You may also find the [`compareLineOutput`](https://github.com/sindresorhus/ava/blob/master/test/helper/compare-line-output.js) helper useful. Here's an example: https://github.com/sindresorhus/ava/blob/7dd7055d360569b937515e907a5af22df07cbb22/test/reporters/mini.js#L268:L281